src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const allSpans = document.querySelectorAll('.population');
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+let totalValue = 0;
+
+for (let i = 0; i < allSpans.length; i++) {
+  let arr = [];
+
+  arr = allSpans[i].textContent.split(',');
+
+  const value = arr.join('');
+
+  totalValue += +value;
+}
+
+const averageValue = Math.ceil(totalValue / allSpans.length);
+
+const totalValueStr = totalValue.toString().split('');
+const averageValueStr = ('' + averageValue).split('');
+
+const resTotal = [
+  ...totalValueStr.slice(0, 1),
+  ',',
+  ...totalValueStr.slice(1, 4),
+  ',',
+  ...totalValueStr.slice(4, -3),
+  ',',
+  ...totalValueStr.slice(-3),
+];
```

Comment:
The logic for formatting `totalValue` assumes that the number will always have at least 7 digits. If `totalValue` has fewer digits, this will result in incorrect formatting. Consider dynamically inserting commas based on the length of the number.